src/screens/authentication/UsernameRequest/index.js

```diff
@@ -62,31 +97,31 @@ const UsernameRequest = (props) => {
   }
 
   return (
-    <Background>
+    <>
+      <Icon />
       <KeyboardAvoidingView style={styles.container}>
-        <Image
-          style={styles.drCannabisIcon}
-          source={DrCannabis}
-        />
-        <Text fontVariant='description' style={styles.information}>Antes de iniciar sesión vamos a necesitar un nombre para tu usuario.</Text>
-        <Text fontVariant='h1' style={styles.usernameTag}>Usuario</Text>
+        <Subtitle style={styles.title}>Creá tu nombre de usuario</Subtitle>
+        <Description
+          gray
+          style={styles.description}
+        >
+        Escribí un nombre de usuario para identificarte dentro de la plataforma
```

Comment:
Wrong indentation here.

Reply:
Standard did not mark it. Should I put the text two spaces to the right?

Reply:
Yes, I like increasing indentation inside every tag. I don't know if standard has a rule for that.